### ISSUE TITLE
[ repo ] [generated_edn] Renamed to generated_json ?

### ISSUE THREAD

**Issue by rprimus** [yqrashawn/GokuRakuJoudo#64]
Thu May 28 13:36:54 BST 2020

Hi.

In this repo shouldn't  `GokuRakuJoudo/resources/configurations/generated_edn/` (and the files contained within) have suffix `json` - as these are the files generated for use with Karabiner?

Cheers,

**Comment by yqrashawn** [comment]
Hi Those are edn files in a middle state (about to be converted to json) using in  tests. They are still edn files. What do you mean that they have suffix `json`?

**Comment by rprimus** [comment]
Fri May 29 11:23:41 BST 2020

HI.

Didn't realise these files were also valid `edn` files.  (I found them while searching the repo for `complex_modifications`.)

> What do you mean that they have suffix json?

I was asking - "Shouldn't the files have a `.json` suffix?"  The original question was too wordy and should have been broken up into smaller pieces.

As the files are valid `edn` files, 

Thanks for creating this project!
